eland/operations.py

```diff
@@ -584,9 +584,58 @@ def aggs_groupby(
         if is_dataframe_agg:
             # Convert header columns to MultiIndex
             agg_df.columns = pd.MultiIndex.from_product([headers, pd_aggs])
+        else:
+            # Convert header columns to Index
+            agg_df.columns = pd.Index(headers)
```

Comment:
Can we add a testcase that verifies this behavior?

Reply:
Yes :)